src/routes/chat/components/message/Message.js

```diff
@@ -0,0 +1,74 @@
+import React, { memo } from 'react';
+import useStyles from 'isomorphic-style-loader/useStyles';
+import PropTypes from 'prop-types';
+import classNames from 'classnames';
+// After creating user model will use Avatar component instead <img>
+// import Avatar from '../avatar';
+import IconMessageStatus from '../messageStatus/IconMessageStatus';
+
+import s from './Message.css';
+
+const testIsRead = true;
+
+function Message({ isMe, createdAt, text, user }) {
+  useStyles(s);
+  return (
+    <div
+      className={classNames('message', {
+        [s.isMe]: isMe,
+      })}
+    >
+      <div className={s.message}>
+        <span
+          className={classNames('', {
+            [s.fullname]: isMe,
+          })}
+        >
+          {user.fullname}
+        </span>
+        <div>
+          <div className={s.content}>
+            <img
+              alt="User"
+              className={s.avatar}
+              src="https://googleretailtraining.exceedlms.com/assets/student/google/default_avatar-5298a7fcd9a9a0ce0d2b32e9aa826a32cc521cb5249540190792626b75504b39.png"
+            />
+            <div
+              className={classNames(`${s.bubble}`, {
+                [s.bubbleIsMe]: isMe,
+              })}
+            >
+              <p>{text}</p>
+            </div>
+            <IconMessageStatus isMe={isMe} isRead={testIsRead} />
+          </div>
+          <span
+            className={classNames(`${s.date}`, {
```

Comment:
Is there any reason to wrap dynamic classes with `${}`?

Reply:
Agree, I change it.

Reply:
ok, do please